src/content/documentation/develop/porting-a-legacy-firefox-extension.md

```diff
@@ -170,4 +170,5 @@ You can use the links [here](https://developer.mozilla.org/docs/Mozilla/Add-ons#
 
 <!-- END: Single Column Body Module -->
 
-
+[/documentation/develop/comparison-with-the-add-on-sdk]:/documentation/develop/comparison-with-the-add-on-sdk
+[/documentation/develop/comparison-with-xul-xpcom-extensions]:/documentation/develop//documentation/develop/comparison-with-xul-xpcom-extensions-xpcom-extensions
```

Comment:
This is a bad link (see the duplication of `xpcom-extensions` and `/documentation/develop`) . Also, where possible avoid using paths for references (if it's easy to confuse what's the link and what's the ref it makes maintenance harder).